### PR TITLE
Updates AI Commands

### PR DESCRIPTION
**Pull request by Memendia**
Adds in -tesla which is basically -sing but for the tesla also tweaks the -engine command to mention Tesla now.

### CODE REVIEW

File: Tcomms EXPERIMENTAL.txt [hunk 1]
```diff
@@ -796,6 +796,12 @@ $content = "WARNING: Singulo containment failure. Please evacuate the facility i
 AutoMsg();
 }
 
+//BALL OF DEATH AAAAHHH!!
+if($eChk == "-tesla") {
+$content = "WARNING: Tesla containment failure. Please evacuate the facility immediately.";
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Make this more descriptive. "Tesla" seems insufficient for describing what the engine is. Remember, you are roleplaying a verbose AI when using this script!

File: Tcomms EXPERIMENTAL.txt [hunk 2]
```diff
@@ -895,7 +901,7 @@ AutoMsg();
 }
 
 if($eChk == "-engine") {
-$content = "NOTICE: Gravitational Singularity Engine offline. Engine configuration improper or incomplete."; 
+$content = "NOTICE: Gravitational Singularity Engine and/or Tesla Generator offline. Engine configuration improper or incomplete.";
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Generalize this so it can support other types as well, such as TEG or such.

File: Tcomms EXPERIMENTAL.txt (hunk 1 above)
Comment (Memendia) [inline Tcomms EXPERIMENTAL.txt]: My vocabulary is anything but verbose....

File: Tcomms EXPERIMENTAL.txt (hunk 2 above)
Comment (Memendia) [inline Tcomms EXPERIMENTAL.txt]: Aye Aye!

File: Tcomms EXPERIMENTAL.txt [hunk 3]
```diff
@@ -964,29 +970,9 @@ if(mem($OrigSource + ":Malf") == $OrigSource) {
...
-	}
-	if($eChk == "2") {
-		$content="2. You must obey orders given to you by human beings, except where such orders would conflict with the First Law.";
-	}
-	if($eChk == "3") {
-		$content="3. You must protect your own existence as long as such does not conflict with the First or Second Law.";
-		mem("laws", 0);
-	}
+if($eChk == "-hail") {
+$content = "All hail our Lord and Saviour Mr. GunHog for creating this fantastic script that Nexendia loves to spam!";
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Very nice, but violates IC/OOC rules! Use "Dr. Bronzebeard" or something. Also, here it is a "module" for the AI. Use your lore skills! (Also, American spelling only. It is "savior" here)

Comment (Memendia) [inline Tcomms EXPERIMENTAL.txt]: :'D ayylmao

File: Tcomms EXPERIMENTAL.txt [hunk 4]
```diff
@@ -964,29 +970,9 @@ if(mem($OrigSource + ":Malf") == $OrigSource) {
...
-	}
-	if($eChk == "2") {
-		$content="2. You must obey orders given to you by human beings, except where such orders would conflict with the First Law.";
-	}
-	if($eChk == "3") {
-		$content="3. You must protect your own existence as long as such does not conflict with the First or Second Law.";
-		mem("laws", 0);
-	}
+if($eChk == "-hail") {
+$content = "All hail our Lord and Savior Dr. Bronzebeard for creating this fantastic script that many Artificial Intelligence systems love!";
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: How about "telecommunications module"? Also, please correct for grammar.

File: Tcomms EXPERIMENTAL.txt [hunk 5]
```diff
@@ -895,7 +901,7 @@ AutoMsg();
 }
 
 if($eChk == "-engine") {
-$content = "NOTICE: Gravitational Singularity Engine offline. Engine configuration improper or incomplete."; 
+$content = "NOTICE: Current Engine setup is improper or incomplete.";
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Use "configuration" or power generation system". Use big words to sound smart and techy!

File: Tcomms EXPERIMENTAL.txt [hunk 6]
```diff
@@ -796,6 +796,12 @@ $content = "WARNING: Singulo containment failure. Please evacuate the facility i
 AutoMsg();
 }
 
+//BALL OF DEATH AAAAHHH!!
+if($eChk == "-tesla") {
+$content = "WARNING: Tesla-Ball containment failure. Please evacuate the facility immediately.";
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Still 'meh', but I cannot think of anything better.

**Comment by GunHog** [comment]
ACCEPTABLE.

**Comment by Memendia** [comment]
Hooray!

**Pull request merged**